command/container-show.go

```diff
@@ -67,6 +67,7 @@ func (c *ContainerShowCommand) Run(args []string) int {
 			tbl += fmt.Sprintf("Memory limit:\t%d MB\n", co.Limits.Memory)
 			tbl += fmt.Sprintf("CPU limit:\t%d MHz\n", co.Limits.CPU)
 			tbl += fmt.Sprintf("Network limit:\t%d Mbps\n", co.Limits.Net)
+			tbl += fmt.Sprintf("Enabled capabilities: \t%s \n", strings.Join(co.DockerCapabilities, ", "))
```

Comment:
no space just comma (even if it is more readable, this could avoid a lot of parsing/potential issues